src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -974,6 +977,12 @@ public override object VisitField(FieldSymbol symbol, TypeCompilationState argum
                 // Any errors generated below here are considered Emit diagnostics 
                 // and will not be reported to callers Compilation.GetDiagnostics()
 
+                ImmutableArray<SourceSpan> dynamicAnalysisSpans = ImmutableArray<SourceSpan>.Empty;
+                if (_moduleBeingBuiltOpt.EmitOptions.EmitDynamicAnalysisData)
+                {
+                    flowAnalyzedBody = Instrumentation.InjectInstrumentation(methodSymbol, flowAnalyzedBody, methodOrdinal, compilationState, _compilation, _diagnostics, _debugDocumentProvider, out dynamicAnalysisSpans);
```

Comment:
My real concern is that the trees presented to the instrumentation logic appear to be unlowered. @AlekseyTs has explained that that is simple to do. I'll open an issue to track this.